### ISSUE TITLE
Admin function: Add organization orbats

### ISSUE THREAD

**Issue by 132ndNeck** [birgersp/dcs-bajas#36]
Would it be possible to add in a admin function that defines what the different organizations:

For example:
Armor Platoon: 4 MBTs
Armor Company: 3 Platoons (12 MBTs)
Armor Battalion: 3 Companys (36 MBT's)

Artillery platoon: ( 3 Artillery pieces)
Artillery Company: 6 Artillery pieces)
Artillery Battaltion (18 Artillery pieces)

Etc..

**Comment by jkhoel** [comment]
@birgersp would it be possible to make some sort of shared setup-file so that you could define this and then have it imported to both MINT and AutoGFT?

**Comment by birgersp** [comment]
Yes, I'll make a template for how you can define these organizations. I'm thinking it should be a javascript file used by the server, so it probably won't be directly implementable into AutoGFT.

**Comment by birgersp** [comment]
Though I suggest that this shouldn't be a "admin function" (as a per-user-feature), but merely a single definition used by the server. Is there any point in having multple definitions (customizable per user)?

**Comment by 132ndNeck** [comment]
@birgersp It might been a bad explanation from me, but what I meant with admin function, was a single defintion used by the server. Just to make sure that everyone have the same information presented.
My intention would be to produce a intelligence document for a campaign, which contains the details of the orbat.

I dont see a big point of having it as customizable per user, as I think it would be important in MP that everyone have the same information? (on unit sizes etc..) So that we dont come in a situation where player 1 have a platoon as 4 tanks, where player 2 has a platoon as 8 tanks for example

HEre is a example of such a intel document which I made for our current ongoing campaign:
https://www.dropbox.com/s/y9x34ddyaqwry55/2015-11-30%20OP%20Georgian%20Thunder%20-%20Background%20and%20Intel.pdf?dl=0

Once I am back , and MINT and Autogft are in a workable state , then I plan to create a new campaign, creating new intel, updates, situation to make it fit in this framework, and use MINT activly for that campaign. Starting with running for a weekend, with people having the ability to jump in and fly at any time, but also creating events as we do now, where we try to have as many people as possible to attend at the same time, using MINT for preperations and taskings.
